CRM/Sqltasks/Upgrader.php

```diff
@@ -242,4 +242,25 @@ public function upgrade_0100() {
     return TRUE;
   }
 
+  /**
+   * Update 'name' column in 'civicrm_sqltasks' table if column does exist
+   * Sets max length to 255
+   *
+   * @return bool
+   * @throws \Exception
+   */
+  public function upgrade_0110() {
+    $this->ctx->log->info('Change character limit(set 255) for \'name\' column in \'civicrm_sqltasks\' table.');
+    $isColumnExists = CRM_Core_DAO::singleValueQuery("SHOW COLUMNS FROM `civicrm_sqltasks` LIKE 'name'");
```

Comment:
`$isColumnExists` and the `if` is not necessary as the `name` column always exists.